.coderabbit.yaml

```
@@ -0,0 +1,9 @@
+language: ko
+reviews:
+  enabled: true
+  branches:
+    include:
+      - main
+      - develop
+      - feature/*
+  review_status: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)